src/components/SignName.vue

```diff
@@ -10,6 +10,7 @@
           label="Full Name"
           placeholder="John Doe"
           required
+          @keyup="checkFormFilled"
```

Comment:
This works as intended, but I was thinking we could clean up some of the repeated code if we used a watcher on the form property. I tried a quick and dirty test and it works how I think it should. There is another watcher in `SignName.vue` that you could use as an example. The implementation would be essentially the same.

https://vuejs.org/guide/essentials/watchers.html

Reply:
Thank you for this suggestion, I will look into it and make the necessary updates.